README.md

```diff
@@ -81,82 +81,40 @@ Please note that this package is still under active development and the API is s
 
 ## Config Options
 
-| Name          | Type     | Description                                                                   | Options                              | Required | Default  |
-| ------------- | -------- | ----------------------------------------------------------------------------- | ------------------------------------ | -------- | -------- |
-| `scope`       | `string` | User flow type                                                                | `['create', 'update']`               | No       | 'create' |
-| `data`        | `object` | Data passed to the API                                                        | Content depends on value of `scope`  | Yes      | none     |
-| `accessToken` | `string` | API [OAuth token](https://developers.arcadia.com/#operation/createOAuthToken) |                                      | Yes      | none     |
-| `env`         | `string` | API environment                                                               | `['local', 'sandbox', 'production']` | Yes      | none     |
-| `client`      | `string` | Name used to reference organization in app                                    |                                      | Yes      | none     |
-| `callbacks`   | `object` | Callback functions                                                            |                                      | No       | none     |
-| `uiTheme`     | `string` | UI color theme                                                                | `['light', 'dark']`                  | No       | 'light'  |
+| Name                | Type     | Description                                                                   | Options                              | Required | Default |
+| ------------------- | -------- | ----------------------------------------------------------------------------- | ------------------------------------ | -------- | ------- |
+| `accessToken`       | `string` | API [OAuth token](https://developers.arcadia.com/#operation/createOAuthToken) |                                      | Yes      | none    |
+| `env`               | `string` | API environment                                                               | `['local', 'sandbox', 'production']` | Yes      | none    |
+| `newCredentialData` | `object` | Data used to prefill new credentials                                          | Content depends on value of `scope`  | No       | none    |
```

Comment:
We should remove this string from the Options column: `Content depends on value of `scope`. We no longer have the concept of `scope`. 

Perhaps in the Description column we should note that this data is only used when creating new credentials and will otherwise throw an error if used with a UtilityConnectToken associated with exiting credentials?